src/colmap/estimators/cost_functions.h

```diff
@@ -401,6 +401,33 @@ struct AbsolutePosePositionPriorCostFunctor
   const Eigen::Vector3d position_in_world_prior_;
 };
 
+// 3-DoF error on the camera orientation in the world coordinate frame.
+struct AbsolutePoseRotationPriorCostFunctor
+    : public AutoDiffCostFunctor<AbsolutePoseRotationPriorCostFunctor, 3, 4> {
+ public:
+  explicit AbsolutePoseRotationPriorCostFunctor(
+      const Eigen::Quaterniond& prior_rotation_cam_from_world)
+      : prior_rotation_world_from_cam_(
+            prior_rotation_cam_from_world.conjugate()) {}
```

Comment:
nit: a bit easier to read
```suggestion
            prior_rotation_cam_from_world.inverse()) {}
```